Fix typo

tpraxl opened the pull request.

UniversalSuperBox commented:
Hi @tpraxl, are you able to make this fix so we can merge this?

Thank you again!

tpraxl commented:
I fixed this using the edit button in github. I see no way to change this PR without cloning, editing locally, pushing… sigh.. I added a new PR. Please see #252 and close this one.

UniversalSuperBox commented:
Superseded by #252
